package.json

```diff
@@ -111,7 +111,7 @@
     "silent-error": "^1.0.0",
     "symlink-or-copy": "^1.0.1",
     "temp": "0.8.3",
-    "testem": "0.9.11",
+    "testem": "testem/testem",
```

Comment:
Moving to master will make it difficult for repeatable builds. Not a fan of this.